### PR TITLE
HPCC-13932 Add version check when connecting to Cassandra

### PR DESCRIPTION
**Pull request by richardkchapman**
Signed-off-by: Richard Chapman rchapman@hpccsystems.com

### CODE REVIEW

**Comment by richardkchapman** [comment]
@jamienoss Please review

File: plugins/cassandra/cassandrawu.cpp [hunk 1]
```diff
@@ -3327,6 +3370,50 @@ class CCasssandraWorkUnitFactory : public CWorkUnitFactory, implements ICassandr
...
+                return rowToPTree(NULL, NULL, versionMappings, row);
+        }
+        catch (IException *E)
+        {
+            EXCLOG(E);
+            E->Release();
+        }
+        catch (...)
+        {
+            DBGLOG("WARNING: Unknown exception caught");
```
Comment (jamienoss) [inline plugins/cassandra/cassandrawu.cpp]: @richardkchapman do you think this should be more informative?

**Comment by jamienoss** [comment]
@richardkchapman only the one comment.

File: plugins/cassandra/cassandrawu.cpp (hunk 1 above)
Comment (richardkchapman) [inline plugins/cassandra/cassandrawu.cpp]: Hopefully it will never happen. But I can add a bit more text

**Comment by richardkchapman** [comment]
@jamienoss Repushed with improved wording

**Comment by jamienoss** [comment]
@richardkchapman merge away.

**Pull request merged**